.travis.yml

```diff
@@ -5,8 +5,8 @@ services:
 
 language: go
 go:
-  - "1.5.x"
   - "1.10.x"
+  - "1.11.x"
```

Comment:
The lower of this two entries is to check the minimum toolchain we support (mostly for distro packaging and LTS products). I think after this PR the new minimum is 1.7.x. Can you please update both this and the README in that sense?

Reply:
Sure, all updated and pushed. I kept 1.11 in because it makes sense to test the latest stable as well.

Reply:
Yup. You can drop 1.10 too if you feel like it.

Reply:
I'd like to keep N-1 please. It's what I and many people who aren't on the latest distro still use. I drop the versions when there's no stable Fedora version using them anymore. (My personal rule of thumb anyways.)